coltune_analyze.py

```diff
@@ -255,56 +255,56 @@ def writeResult(num_rank_list, coll_result, outfil):
     WIDTHS = [10,        12,             15,               20,             15,              20,             15]
     f = open(outfil, "w")
 
-    print >> f, ""
+    print("", file=f)
```

Comment:
Can you post example outputs so that I can confirm nothing has broken?

Reply:
I sent you some example output via mail for confirmation as I cannot preserve the white spaces here in the comments.

Reply:
Thanks, looks fine to me - are you sure you can't preserve whitespace? Are you using markdown? Looks fine to me.

```
#Nranks    Message_size Best_Algorithm  Best_Latency         Ref_Algorithm   Ref_Latency          Speedup         
2          1            1               0.12(0.01)           0               0.14(0.02)           1.16           
2          2            5               0.14(0.01)           0               0.15(0.01)           1.02           
2          4            3               0.14                 0               0.14(0.01)           1.02           
2          8            1               0.13(0.01)           0               0.14                 1.05           
2          16           1               0.13(0.01)           0               0.13(0.01)           1.05           
2          32           1               0.14(0.02)           0               0.14(0.01)           1.02           
2          64           0               0.14(0.01)           0               0.14(0.01)           1.00           
2          128          1               0.19(0.02)           0               0.20(0.02)           1.05           
2          256          1               0.19(0.01)           0               0.20(0.01)           1.07           
2          512          1               0.22(0.01)           0               0.23(0.03)           1.05           
2          1024         0               0.24(0.01)           0               0.24(0.01)           1.00           
2          2048         1               0.28(0.01)           0               0.35(0.03)           1.25           
2          4096         1               0.38(0.03)           0               0.50(0.03)           1.32           
2          8192         4               0.64(0.05)           0               0.76                 1.19           
2          16384        4               1.12(0.15)           0               1.80(0.34)           1.61           
2          32768        4               2.19(0.21)           0               2.84(0.50)           1.30           
2          65536        4               3.29(0.02)           0               5.01(0.36)           1.52           
2          131072       4               6.19(0.05)           0               9.34(0.28)           1.51           
2          262144       4               10.67(0.76)          0               19.15(1.03)          1.79           
2          524288       7               27.40(0.05)          0               37.77(0.08)          1.38           
2          1048576      1               51.00(0.41)          0               76.33(0.42)          1.50           

```